src/components/Form/Textarea/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { styled } from 'stitches.config';
+import React, { HTMLAttributes } from 'react';
+import Label from '@components/Form/Label';
+import HelpMessage from '@components/Form/HelpMessage';
+
+interface TextareaProps extends HTMLAttributes<HTMLTextAreaElement> {
+  label: string;
+  value: string;
+  message?: string;
+  required?: boolean;
+  maxLength?: number;
+}
+
+const Textarea = React.forwardRef<HTMLTextAreaElement, TextareaProps>(
+  ({ label, message, required, ...props }: TextareaProps, ref) => {
+    return (
+      <SContainer>
+        <Label required={required}>{label}</Label>
+        {message && <HelpMessage>{message}</HelpMessage>}
+        <STextarea ref={ref} {...props} />
+        {props.maxLength && (
+          <STextCount overflow={props.value.length > props.maxLength}>
```

Comment:
```suggestion
          <STextCount overflow={props.value.length >= props.maxLength}>
```
피그마 보니까 300/300일 때 빨간 글씨 떠야 하는 것 같아요!

Reply:
아하 그렇네요. 근데 요거는 먼저 인우랑 얘기해보고 의도를 한번 파악해볼게요!(300자 제한이면 300자까지 쓸 수 있어야 되는 것 같은데, 빨간 글씨가 뜨면 299자만 써야되는 것 같아서 뭔가 헷갈리네요 ㅎ)

Reply:
넵 좋아요 !!